deel-2-flexbox-layout/opdracht-3/styles/styles.css

```diff
@@ -1 +1,47 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+
+body{
+    display: flex;
+    flex-direction: row;
+    min-height: 100vh;
+}
+
+.col-2{
+    display: flex;
+    flex-direction: column;
+    flex-basis: auto;
+    flex-grow: 1;
+    flex-shrink: 1;
+}
+
+main{
+    display: flex;
+    flex-direction: row;
+    flex-basis: auto;
+    flex-grow: 1;
+    flex-shrink: 1;
+}
+
+.col-1{
+    flex-basis: 15vw;
+}
+
+header{
+    flex-basis: 90px;
+}
```

Comment:
Officieel gezien mocht de header niet groeien of krimpen. Dus horen er nog 2 properties schrik & grow hierbij met value van 0.